usaspending_api/search/v2/views/search.py

```diff
@@ -344,3 +346,60 @@ def post(self, request):
 
             else:  # recipient_type
                 raise InvalidParameterException('recipient type is not yet implemented')
+
+
+class SpendingByGeographyVisualizationViewSet(APIView):
+
+    def post(self, request):
+        """Return all budget function/subfunction titles matching the provided search text"""
+        json_request = request.data
+        scope = json_request.get('scope', None)
+        filters = json_request.get('filters', None)
+
+        if scope is None:
+            raise InvalidParameterException('Missing one or more required request parameters: scope')
+        if filters is None:
+            raise InvalidParameterException('Missing one or more required request parameters: filters')
+        potential_scopes = ["recipient_location", "place_of_performance"]
+        if scope not in potential_scopes:
+            raise InvalidParameterException('scope does not have a valid value')
+
+        # build sql query filters
+        queryset = transaction_filter(filters)
+
+        # define what values are needed in the sql query
+        # queryset = queryset.values('action_date', 'federal_action_obligation')
```

Comment:
let's remove this if it's not being used.

Reply:
This will be commented back in for the speedup branch.